examples/MNIST_scikit_saliency.ipynb

```diff
@@ -0,0 +1,544 @@
arUhM+hkH1q9nraajujJO63H57Aw1WGe2+Ned9rj9UnGzw1ENUoES+ynFRfoygdGTmeNtqv46jW6AORWuuquIub7EI9zXSecc8EYQrAIeRcmCc0AyUadRkkn3e+blWAbdD10xOXI5BbBFTQ2IwwmXTnpYZoZN51W1qFN7K9jJ/bGw7LxXe86TbQNceHUe96Z2OSVgEjMK4khYznEyE+3/vPpV9VprpSDeqEVYJUlL3KsESZZgElO7QN3hhlv3R4BUFj1+OTWsDVYbbwG1lz76sUzeXw62QYA13v9oPstmCPmjJ7x80zeaCtQ2A/3nvPcc1btntOQjJ8bRj88ZWIM3hoNA7Q5myHoRN+pMLfPj8ZP2/27b/Bc10nnhehXDhG/0K0WjHGrL7jN1TM0iB2o2zh72w08Gj44t6Y2fKg8f+S6h1deJmeJ3N748Y73epCNfn/GaCmXHH6frh0i/mSxtYsBvFu9JrfrlYvOaGhysxfddZ+KThLS6aP5VahBs2HsHNlXSw6p2Ou/9jlyDaep4OI5zz0qJHbW47Uk4+e60tch6d2cAZ0oAkCM/WqioIYIWk/OZhVwWyHW9YAiNSI9Ef2xd7m6mb3XGHrMiwh+lQzdJosCK3ezsbpDk55U/DkmF15XBDXabZw6nYmTolkl/7chL4+s9ZhT414XVtdTwbYx1y1xBP3yflCPZLzHuyqw3nM74cc4X1kx5EZD6j7A0Acab9f6ztnO+NELcRMux1EhnUPP/TBjRMx6QeIJq+BC8BqePOw9jild3/zMz+O4cNd5HO8wkvFznTnmxrayajNhuq2uHNp350b13zpv+pNMG183HsLKDd0miYZNTZSWTXG8ZOg8l1ZEz4gaNhb9w8rmLnL9htZ8EXlmsbWbyc6JZ2nDuTF7WVzvnvb1+o6qr6XUM35Aw1iuN9Hc5OrFF+IEicdH9VlrMRvHuE5Dn4yfG0y7QPQ1FQ6bWNKkc870J6QYKth8GDZKqJ/jYk4cTWsACZrA7IPm9xyqcRjv8f6mwJDu/VPRD4G1t2swhG6i8awJQbbhsR4nHe+bdlnOEgI6T55XeSVHPH5Sjqr2epHjHUUyfm44m5PM5k4gTfrnyHNKo4/9cidvz6lp791ND6cPslbhtUm/4gvWJ9D0fXgOh3k6Q6zqan8/6vn9fxOHcpYQ0HG86P18VLjqrJ/vZR+vTzJ+Emukyf094pwm+eT1eT6b97RWBqUb/aXzkg2adAUTL0Iyfq4BaQG8WNL4JxLnSzJsTsZRYajzCotd5+MlH2MikUgkEteA8w6LXefjJeMnkUgkEokbwHl71K7y8ZLxk0gkEonEFeS0xsLzPCQ36XhyXNVD4vIgIg+B373oz3GFeTOE8MpZX5zG/4VJ439xpLG/WNL4XyyHjn8yfhKJRCKRSNwoUtgrkUgkEonEjSIZP4lEIpFIJG4UyfhJICKvishPiMhnReTTIvIzIvIREXlLRH79Jb3nJ0TkoYj8avz5iy/jfa4CFzH+8X3/nIh8RkR+Q0T+/ct6n8vOBd3//7h37/+OiOy+jPe57FzQ2H9ARH5RRP6XiPxvEfnTL+N9rgIXNP5vish/i2P/SyLyxst4n+eRRA5vOKJa/v8J+LEQwnfFx74OeB/wxZf89v8hhPDXX/J7XGouavxF5MPA9wN/NITwRETuv6z3usxc1PiHED7Z+wx/A/iGl/Vel5ULnHv+PvCTIYR/JSJfA/wM8NZLfL9LyQWO/z8E/m0I4cdE5NuAHwS+5yW+36Ekz0/ijwFVCOFftw+EEH4thPDL/SfFncAvi8j/jD9/JD7+moh8Ku5gf11EvllErIj8aPz7/4jIJ0kcxUWN/18C/kUI4Ul8zwcv8RwvM5fh/v9u4MfP/cwuPxc19gHYjr/vAL/3ks7vsnNR4/81wC/E338R+DMv6fyOJXl+Eh8HPn2C5z0A/mQIYRG9Bj8O/EHgzwM/F0L4ByJigRHw9cDrIYSPA4jIrSOO+WdF5FuA3wE+GUJ42Z6my8hFjf9H4v/9d8ACPxBC+NkXPJeryEXe/4jIm8AHWS0GN4mLGvsfAH4+etzGwJ94wfO4qlzU+P8a8J3APwW+A9gSkbshhEcveD6nIhk/iZOSAf9cRL4eaIiLJ/A/gB8RkQz46RDCr4rI54APicg/A/4r8POHHO+/AD8eQliKyF8Bfgz4tpd9EleY8x5/B3wY+FbgDeBTIvK1IYTdl3oWV5fzHv+W7wL+YwiheXkf/cpz3mP/3cCPhhD+kYh8E/DvROTjIWy2nE9Eznv8/0483ieATwFfisd9T0lhr8RvAH/gBM/7JPAV4OtQqz8HCCF8CvgW9Ab+URH53hhK+Trgl4C/Cvzw5sFCCI9CCMv45w+f8DNcRy5k/IG3gf8cQqhCCP8P9b59+MVO5UpyUePf8l3czJAXXNzY/wXgJ+MxfgUogHsvciJXlIua+38vhPCdIYRvAP5efGz3RU/mtCTjJ/ELwEBE/nL7gIj8fhH55o3n7QBfjruj70FDJa3b/ishhB9Cb/RvFJF7gAkh/BSaXPiNm28qIq/1/vx24DfP8ZyuEhcy/sBPo14f4vM/AnzuHM/rqnBR44+IfBS4DfzKOZ/TVeGixv4LwB+Px/gYavw8PNczuxpc1Nx/T0Ra2+P7gR855/M6ESnsdcMJIQQR+Q7gn4jI3wUWwOeBv7Xx1H8J/JSIfC/ws8A0Pv6twPeJSAUcAN8LvA78m40bfJO/KSLfDtTAY+AT53RKV4oLHP+fA/6UiHwGdTl/33sdc78MXOD4g3p9fiLcUJn9Cxz7vw38UEzGDcAnbuI1uMDx/1bgB0UkoGGvv3ZOp3QqUnuLRCKRSCQSN4oU9kokEolEInGjSMZPIpFIJBKJG0UyfhKJRCKRSNwokvGTSCQSiUTiRpGMn0QikUgkEjeKZPwkEolEIpG4USTjJ5FIJBKJxI0iGT+JRCKRSCRuFP8fSAvsxam7CvoAAAAASUVORK5CYII=\n",
```

Comment:
Can you check these output saliency maps? Somehow the saliency maps for all digits seemed to have collapsed into a single output, which is different from your previous commits. I couldn't find any obvious changes with the `app` function itself, so it may just need an output refresh.

Reply:
Not sure how I missed that... it's fixed now

Reply:
Not sure how I missed that... it's fixed now